README.md

```diff
@@ -20,3 +20,29 @@ randomly without memory instead of systematically trying them?
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+
+## Answer 
+We know the best-case scenario for this code would be _O_(n) and that's if the array is already sorted when you put it in. It would just have to look at each element in the array and since it will be sorted the runtime will be based on the amount of elements there are.
+
+The worst-case scenario is infinite time or _O_(n!). The reason is because it will go on forever and there might not be an end to the sorting and even when it is sorted it might not be correct either. With this, it might never stop sorting because it has no way to see that it is sorted completely and is going to keep on going until it eventually sorts it. Which might be infinite. 
+
+The number of permutations is n! because we get the length of the array, which is n, and then it goes down the list of the arrays multiplying one another, which gives you n factorial. This is in the GenPerm function. 
+
+The work being done is $O(n)$ because the isSorted function checks if each permutation is sorted. This function iterates over the array once, comparing each adjacent pair of elements. 
+
+With those added together, you would get $O(n + n!)$. This would give us $O(n!)$ as the worst-case scenario for this function. 
+
+**Response to Questions:**
```

Comment:
Please adjust your answers above. If you're first saying one thing and then another, it makes it really difficult for me to figure out what you mean.